data/_models/shared/Resources.csv

```diff
@@ -1,208 +1,217 @@
-Resources,,outcomes,text,,,,,,,,,A short description of the study outcomes,EMA
-Resources,,results,text,,,,,,,,,A brief summary of the results of the study on study completion (e.g. from abstract),"DataCatalogueFlat,EMA,StudiesStaging"
-Resources,,topic,ontology_array,,,CatalogueOntologies,Study topics,,,,,An initial classification of the study purpose,EMA
-Resources,,topic other,text,,,,,,,,,"If the study is not concerning any of the proposed categories, please specify the details",EMA
-Resources,,trial regulatory scope,ontology_array,,,CatalogueOntologies,Study trial regulatory scopes,,,,,Classification of the clinical trial in relation to the medicines authorisation,EMA
-Resources,,study design classification,ontology_array,,,CatalogueOntologies,Study design classification,,,,,Study design classifications,TODO Merge with resource type
-Resources,,study design classification other,text,,,,,,,,,Further details on design,EMA
-Resources,,study scope,ontology_array,,,CatalogueOntologies,Study scopes,,,,,Scope of the study,EMA
-Resources,,study scope other,text,,,,,,,,,If scope 'other',EMA
-Resources,,theme,hyperlink_array,,,,,,,,dcat:theme,A main category of the resource. A resource can have multiple themes.,DataCatalogueFlat
+tableName,tableExtends,columnName,columnType,key,required,refSchema,refTable,refLink,refBack,validation,semantics,description,profiles,defaultValue
+Resources,,,,,,,,,,,"dcat:DatasetSeries,dcat:Dataset",Resource of data and/or samples. Use 'resource type' to specify,"DataCatalogueFlat,EMA,CohortsStaging,RWEStaging,NetworksStaging,StudiesStaging,UMCGCohortsStaging,UMCUCohorts,INTEGRATE",
+Resources,,overview,heading,,,,,,,,,General information,"DataCatalogueFlat,EMA,CohortsStaging,RWEStaging,NetworksStaging,StudiesStaging,UMCGCohortsStaging,UMCUCohorts,INTEGRATE",
+Resources,,id,,1,TRUE,,,,,,oboInOwl:hasAlternativeId,Internal identifier,"DataCatalogueFlat,EMA,CohortsStaging,RWEStaging,NetworksStaging,StudiesStaging,UMCGCohortsStaging,UMCUCohorts,INTEGRATE",
+Resources,,pid,,2,TRUE,,,,,,dct:identifier,Persistent identifier,"DataCatalogueFlat,EMA,CohortsStaging,RWEStaging,NetworksStaging,StudiesStaging,UMCGCohortsStaging,UMCUCohorts,INTEGRATE",
+Resources,,name,text,3,TRUE,,,,,,dct:title,Name used in international projects,"DataCatalogueFlat,EMA,CohortsStaging,RWEStaging,NetworksStaging,StudiesStaging,UMCGCohortsStaging,UMCUCohorts,INTEGRATE",
+Resources,,local name,,,,,,,,,rdfs:label,"If different from above, name in the national language","DataCatalogueFlat,CohortsStaging,RWEStaging",
+Resources,,acronym,,,,,,,,,dct:alternative,"Acronym, if applicable","DataCatalogueFlat,EMA,CohortsStaging,RWEStaging,NetworksStaging,StudiesStaging,UMCGCohortsStaging,UMCUCohorts,INTEGRATE",
+Resources,,type,ontology_array,,TRUE,CatalogueOntologies,Resource types,,,,dct:type,"Type of resource, e.g. registry, cohort, biobank","DataCatalogueFlat,CohortsStaging,RWEStaging,NetworksStaging,StudiesStaging,UMCGCohortsStaging,UMCUCohorts,INTEGRATE",
```

Comment:
type will have a recommended controlled vocabulary in healthdcat-ap. For health datasets containing personal level information, the type of the dataset MUST take the value "personal data”.

Reply:
healthdcatap:healthCategory